git_machete/client.py

```diff
@@ -18,7 +19,6 @@
     get_parsed_github_remote_url, is_github_remote_url, create_pull_request,
     derive_current_user_login, set_milestone_of_pull_request,
     add_assignees_to_pull_request, add_reviewers_to_pull_request)
-from git_machete.options import CommandLineOptions
```

Comment:
Any reason for that change? 🤔

Reply:
this is connected to how mocking mechanism works. In fact it mocks a namespace. on the other hand `from import` creates a new namespace within current file scope which makes from imported stuff unmockable.